doc/introduction/pennylane.rst

```diff
@@ -39,14 +39,25 @@ through the computation - an essential process in optimization and machine learn
 Learn more
 ----------
 
-In the following sections you can learn more about quantum circuits and interfaces in PennyLane:
+In the following sections you can learn more about the key features of PennyLane:
 
-1. :ref:`Quantum circuits <intro_vcircuits>` shows you how PennyLane unifies and
-simplifies the process of programming quantum circuits with trainable parameters.
+1. :ref:`Quantum circuits <intro_vcircuits>` shows how PennyLane unifies and simplifies 
```

Comment:
```suggestion
1. :doc:`circuits` shows how PennyLane unifies and simplifies 
```

Reply:
(if we stick to page links, rather than reference links, we avoid the weird page scrolling behavior)